website/docs/getting-started/index.mdx

```diff
@@ -76,22 +73,22 @@ interface PostQuery {
   }[]
 }
 
-const postsQueryDocument = `
+const postsQueryDocument = /* GraphQL */ `
```

Comment:
adding `/* GraphQL */` comments helps prettier understand that this is not string but graphql document, so he will prettify